src/calculator.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const calculator = (goal, guess) => {
+  let bulls = 0;
+  let cows = 0;
+
+  for (let i = 0; i < guess.length; i++) {
```

Comment:
Consider using functional programming methods like `forEach` instead of a traditional loop for iterating over arrays.